### ISSUE TITLE
[3.8] PHPUnit fails on PHP 5.3

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#16315]
### Steps to reproduce the issue

Run PHPUnit on PHP 5.3 

### Expected result

Tests run

### Actual result

```sh
Fatal error: Class 'PHPUnit_Util_ErrorHandler' not found
```

**Comment by photodude** [comment]
```php
PHP Fatal error:  Class 'PHPUnit_Util_ErrorHandler' not found in /tests/unit/suites/libraries/joomla/log/stubs/log/inspector.php on line 75
Fatal error: Class 'PHPUnit_Util_ErrorHandler' not found in /tests/unit/suites/libraries/joomla/log/stubs/log/inspector.php on line 75
```

**Comment by photodude** [comment]
Does it run if you exclude the `/tests/unit/suites/libraries/joomla/log/stubs/log/inspector.php` tests?

**Comment by mbabker** [comment]
Right now it seems like this and #16311 are very closely related.  @wilsonge @laoneo and @yvesh have been doing some testing/research on it.

**Comment by laoneo** [comment]
#16342 workarounds the issue.

**Comment by brianteeman** [comment]
closed by #16342
